capellambse/model/crosslayer/cs.py

```diff
@@ -66,7 +67,7 @@ class PhysicalPort(c.GenericElement):
 class PhysicalLink(PhysicalPort):
     """A physical link."""
 
-    linkEnds = c.AttrProxyAccessor(
+    link_ends = c.AttrProxyAccessor(
```

Comment:
I checked and confirmed that with the current Capella version (5.0, 5.2 and 6.0) the `PhysicalLink` can only have 2 elements (`PhysicalPort`s) in their `linkEnds` attribute. Imo I'd say that an individual `Accessor` is not necessary since I've encountered an link attribute with such an implicit length condition only once: Exactly here.